EventVisualisation/DataConverter/include/EventVisualisationDataConverter/ConversionConstants.h

```diff
@@ -0,0 +1,108 @@
+// Copyright CERN and copyright holders of ALICE O2. This software is
+// distributed under the terms of the GNU General Public License v3 (GPL
+// Version 3), copied verbatim in the file "COPYING".
+//
+// See http://alice-o2.web.cern.ch/license for full licensing information.
+//
+// In applying this license CERN does not waive the privileges and immunities
+// granted to it by virtue of its status as an Intergovernmental Organization
+// or submit itself to any jurisdiction.
+
+/// @copyright
+/// © Copyright 2014 Copyright Holders of the ALICE O2 collaboration.
+/// See https://aliceinfo.cern.ch/AliceO2 for details on the Copyright holders.
+/// This software is distributed under the terms of the
+/// GNU General Public License version 3 (GPL Version 3).
+///
+/// License text in a separate file.
+///
+/// In applying this license, CERN does not waive the privileges and immunities
+/// granted to it by virtue of its status as an Intergovernmental Organization
+/// or submit itself to any jurisdiction.
+
+///
+/// \file    ConversionConstants.h
+/// \author  Jeremi Niedziela
+/// \author  Maciej Grochowicz
+///
+
+#ifndef ALIROOT_CONVERSIONCONSTANTS_H
+#define ALIROOT_CONVERSIONCONSTANTS_H
+
+#include <TString.h>
+
+enum ETrackType {
+    Standard,
+
+    KinkMother,
+    KinkDaughter,
+
+    V0NegativeDaughter,
+    V0PositiveDaughter,
+    V0Mother,
+
+    CascadePrimaryMother,
+    CascadePrimaryDaughter,
+    CascadeSecondaryMother,
+    CascadeNegativeDaughter,
+    CascadePositiveDaughter,
+
+    MuonMatched,
+    MuonNotMatched,
+    MuonGhost
+};
+
+const TString gDetector[23] = {
```

Comment:
define a constant for the array size. What's the reason to define this `TString`? Elsewhere in the code `std::string` has been use for the return of such descriptors, maybe this can be simple `const char*` unless there is good reason.

Reply:
Indeed, no reason for TString (remains of the old code).